public/menu.js

```diff
@@ -23,6 +41,20 @@ function setMainMenu() {
     {
       label: 'Edit',
       submenu: [
+        {
+          label: 'Lower Opacity',
+          accelerator: 'CmdOrCtrl+-',
+          click() {
+            mainWindow.setOpacity(minus(mainWindow));
+          }
+        },
+        {
+          label: 'Increase Opacity',
+          accelerator: 'CmdOrCtrl+=',
```

Comment:
Should not this be `CmdOrCtrl++`?

Reply:
Nope as the key is known to js as = not as +. + is a concatination key

Reply:
so its not able to be used. if you wanted it to be plus you would have to do CmdOrCtrl+Shift+=

Reply:
I thought it would have been + at first but it does nothing.

Reply:
Change this shortcut to `CmdOrCtrl+Up`